src/main/webapp/app/pages/userPage/UserPage.tsx

```diff
@@ -309,6 +309,9 @@ export default class UserPage extends React.Component<IUserPage> {
         companyName: this.user.company
           ? this.user.company.name
           : values.company,
+        additionalInfo: this.user.additionalInfo
+          ? this.user.additionalInfo
+          : values.additionalInfo,
```

Comment:
This does not work. When you have updated value in `values.additionalInfo`, it's not overwriting the original.